### DISCUSSION TITLE
Focus Group: Training/Teaching material, Documentation and Website

### DISCUSSION THREAD

**Discussion by timosachsenberg** [OpenMS/OpenMS#7338] [General]
Identification of Training Events in 2024
- [ ] Which are the training events we want to participate in 2024?
ISMB2024 : training in one of CGI events
ASMS2024 : Bioinformatics hub. FLASH* will be taught during ASMS short course and TDP workshop 
- [ ] The OpenMS user meeting where, how, and who?
Around HUPO2024 (20-24 Oct. 2024 in Dresden) So 17 - 18 Oct would be great.
We could contact Steffan Neumann in Halle
- [ ] Who are the key groups that we should interact with (e.g., developers of web apps, educational institutions, enterprise users)? 
Students as future users 
Enterprise users - KNIME users and KNIME people (we will meet in the upcoming meeting). Maybe less than 10% based on statistics, but  it is hard to get statistics on users of OpenMS through KNIME.
Mass Spec company, SCIEX, Thermo … we interact with them regarding new mzML formats (mzMLb and mzNext) as well.
- [ ] Are there specific segments within the community that have unique training needs?
People in emerging subfields such as top down and metabolomics. They may need training for more basics.
- [ ] What strategies can we employ to engage these focus groups effectively (e.g., targeted workshops, partnerships, community events)?
Need more discussion
- [ ] How can we tailor our training offerings to meet the specific needs of these groups?
Need more discussion
- [ ] Which new features or technologies have been introduced in the project that require training?
FLASH series. Let's add more.
- [ ] What are the key topics that should be covered in the training events?
Need more discussion
- [ ] How can we ensure the training content remains up-to-date with the latest project developments?
Literally need more man power.
Sprint for training material and checking the trends.
- [ ] Who are the experts within the community who could lead these training sessions? In general, how to distribute tasks? How to organize projects (e.g., have labeled issues that will be addressed in sprints)
Most likely Kohlbacher lab members.
Some possible outside people: Dave Tabb. Lars Nilse https://github.com/lars20070 

Development of Targeted Training Modules
- [ ] What is the current state of training material? Which can we already address? What is needed to provide those?
We need more updated training material. Sam and Matteo found lots of outdated. We could have a review session.
We have good proteomics tutorials but they are often spread over different repositories in pyOpenMS OpenMS. Not well organized. 
Some are missing like how to use FLASH series.
Good basics on OpenMS, workflow, but mostly general concepts. Need a bit of tutorials for different projects. 
- [ ] How can we structure training content into modular units that can be easily combined or customized?
It is needed. Need more discussion
- [ ] Does it make sense to define core modules that every participant should complete, and what are the elective or advanced modules?
Yes. In particular new areas - top down, metabolomics.
- [ ] How can we design modules to fit different durations (1 hour, a few hours, single day, multiple days)?
Need more discussion
- [ ] What are the key learning goals for each module, and how can we ensure they are met within the allotted time?
Need more discussion
- [ ] How can we incorporate interactive elements (e.g., hands-on exercises in teams of two, quizzes) to enhance learning?
Provide interactive examples via e.g., web hosted jupyter notebook materials.  But this needs lots of knowledge on the science than completing docs.
- [ ] What to use for pyOpenMS training? Experience from Ayesha?
Provide interactive examples via e.g., web hosted jupyter notebook materials. 
- [ ] What mechanisms can we put in place to gather feedback on the training modules? How can we ensure the training content is iteratively improved based on community feedback?
- [ ] Can we record trainings (if possible) to provide asynchronous learning experience?
Formal survey form will be helpful. Maybe ASMS course survey form might be a starting point. They tend to be more informative as years go by..
Website:
- [ ] How can we organize training materials on the website for easy access and navigation?
Need more discussion
- [ ] How to prioritize issues on the website for fixing (e.g., in sprints)
Need more discussion

**Comment by timosachsenberg** [comment]
@KyowonJeong @poshul @tjeerdijk @matteopilz @Ayesha-Feroz  I added you to this focus group for the retreat.

**Comment by KyowonJeong** [comment]
Hi @timosachsenberg  it seems that I cannot edit this page! Could you let me do it?

**Comment by KyowonJeong** [comment]
More on the way.

**Reply by KyowonJeong** [reply]
@poshul , @tjeerdijk , and @KyowonJeong have discussed the points in general. @Ayesha-Feroz and @matteopilz : pleases give more inputs. If you cannot edit this directly, please edit on https://docs.google.com/document/d/1ba9sIxWE8C9p7eBxXXdO5vJCshyBS2_qEdanQsMK_k8/edit?usp=sharing. Please use blue font so I can track the change easily.

**Comment by axelwalter** [comment]
Suggestion: add a "hub" for OpenMS web apps on the website. It could include a short description, screenshot(s) and keywords.

**Comment by matteopilz** [comment]
I have uploaded the major restructure of the docs on my repo: https://github.com/matteopilz/OpenMS-docs
This is mostly reordering files and rewriting the navigation tree in order to support the new structure. Keep in mind, that some files will still output warnings and content may not be correctly linked. E.g. the references for the user tutorial are mostly missing, because I have split the sections into different files.

The next step would be to think about changes in general content and tutorials.
